Example/Backpack/View Controllers/ToastViewController.swift

```diff
@@ -19,7 +19,6 @@
 
 import UIKit
 import Backpack
-import MBProgressHUD
```

Comment:
No idea why this needed to import it too, as importing BPKToast should be enough